nixos/modules/services/search/elasticsearch.nix

```diff
@@ -146,8 +147,8 @@ in {
       path = [ pkgs.inetutils ];
       environment = {
         ES_HOME = cfg.dataDir;
-        ES_JAVA_OPTS = toString ( optional (!es6) [ "-Des.path.conf=${configDir}" ]
-                                  ++ cfg.extraJavaOptions);
+        ES_JAVA_OPTS = toString (optional (!es6) [ "-Des.path.conf=${configDir}" ]
+          ++ cfg.extraJavaOptions);
```

Comment:
Are these formatting changes necessary? It'd be easier to review this with all of them removed / part of a single commit with the title: `elasticsearch7: formatting changes`.

Reply:
My bad, the formatter ran automatically and I didn't catch it.
Let me make it into a separate commit

Reply:
I separated into 2 commits. Let me know if anything else.

Reply:
I don't know enough about the package to approve, sorry.

Reply:
No problems! Thank you for having taken a look!